File: LocationAPI/GetLocationByUserId.py
import requests
import json

userId = "ADD YOUR USER ID HERE"
page = 1
itemsPerPage = 100
customId = "my-custom-location-id"
type = "construction"

url = ("https://location-api.hibirdi.com/user/{}/locations?page={}&itemsPerPage={}&customId={}&type={}"
       .format(userId, page, itemsPerPage, customId, type))

response = requests.request("GET", url)

print(json.dumps(response.json(), indent=2))

File: LocationAPI/GetLocationByUserId.js
let data = null;
const xhr = new XMLHttpRequest();

const userId = "ADD YOUR USER ID HERE";
const page = 1;
const itemsPerPage = 100;
const customId = "my-custom-location-id";
const type = "construction";
const url = "https://location-api.hibirdi.com/users/" + userId +
            "/locations" +
            "?page=" + page +
            "&itemsPerPage=" + itemsPerPage +
            "&customId=" + customId +
            "&type=" + type;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === 4) {
    console.log(this.responseText);
  }
});

xhr.open("GET", url);
xhr.send(data);
